src/hotspot/cpu/aarch64/aarch64_vector.ad

```diff
@@ -5760,67 +5760,62 @@ instruct vroundD(vReg dst, vReg src, immI rmode) %{
 
 // anytrue
 
-instruct vtest_anytrue_neon(iRegINoSp dst, vReg src1, vReg src2, vReg tmp, rFlagsReg cr) %{
+instruct vtest_anytrue_neon(rFlagsReg cr, vReg src1, vReg src2, vReg vtmp) %{
```

Comment:
I think we needn't change the name.

Reply:
Yes I agree, other places use `tmp` for temporary vector registers so I have reverted this. Thanks